src/sentry/workflow_engine/models/data_condition.py

```diff
@@ -74,32 +85,44 @@ def get_condition_result(self) -> DataConditionResult:
 
         return None
 
-    def evaluate_value(self, value: float | int) -> DataConditionResult:
-        # TODO: This logic should be in a condition class that we get from `self.type`
-        # TODO: This evaluation logic should probably go into the condition class, and we just produce a condition
-        # class from this model
+    def get_condition_handler(self) -> DataConditionHandler[T] | None:
         try:
-            condition = Condition(self.condition)
+            condition_type = Condition(self.type)
```

Comment:
should we use the `Condition` StrEnum as the possible choices for `DataCondition.type`?

Reply:
yeah - i was thinking we'll probably want to change the schema around, but it's kinda hard to do so  punting on that for now. right now it's weird cause `condition` is used for `gt`, `lt`, etc but `type` is used for the handler stuff. i'm thinking the `type` field should use the enum, then figure out something to do with the `condition` field (i'm using it to filter / pass configuration info for custom handlers - so maybe the json config base? not sure, so i'm pushing along in other areas until that's a bit more clear since it's so hard to do migrations. happy to take recommendations!!)